src/core/js/views/blockView.js

```diff
@@ -1,16 +1,22 @@
 define([
     'core/js/views/adaptView'
-    ], function(AdaptView) {
+], function(AdaptView) {
 
     var BlockView = AdaptView.extend({
 
+        attributes: function() {
```

Comment:
I don't think it's ever actually instantiated, so AbstractView would have been a better name for it